### ISSUE TITLE
Evaluating inductive inference

### ISSUE THREAD

**Issue by sehyunkwon** [vladan-stojnic/ZLaP#2]
Hello! Thank you for your excellent work.

I enjoyed reading your paper, and I am currently running some experiments with your code.

However, in inductive mode, it seems that the accuracy is being measured using test_targets. i.e, `acc = accuracy(scores, test_targets)`. Is this correct?

Also, could you please check again if the output shape of `do_inductive_lp` is `(test_features.shape[0], num_classes)`?

**Comment by vladan-stojnic** [comment]
Hi,

Thanks for the interest in our work. In both transductive and inductive setups we evaluate on the test set, so using `test_targets` is correct. The difference is that in the transductive setup test images are unlabeled nodes of the graph, while in the inductive setup unlabeled nodes of the graph are images from the training set and we get the predictions for test images by representing them using neighbors from the graph (Eq. (11) from the paper).

I have checked and the output of `do_inductive_lp` is `(test_features.shape[0], num_classes)`. The output of `do_inductive_lp` is `scores` array that was defined with the expected shape [here](https://github.com/vladan-stojnic/ZLaP/blob/89ab14bef8904a9397f8b6a7e5f3e6af2c288b93/zlap.py#L149).

**Comment by sehyunkwon** [comment]
Oh I misunderstood. So you were referring to the nodes that make up the graph Laplacian as 'unlabeled nodes.' Now it's clear to me. Thanks!
